src/seedu/addressbook/data/person/Person.java

```diff
@@ -16,6 +16,9 @@
     private Address address;
 
     private final UniqueTagList tags;
+    private int sequenceNumber;
+    private static int nextSequenceNumber = 0;
```

Comment:
while this is not incorrect, saying that the next sequence number is '0' doesn't match with the variable name. there can be two ways of doing this: naming it `currentSequenceNumber` and set it to 0, or `nextSequenceNumber` and set it to 1.